darts/models/forecasting/tft_model.py

```diff
@@ -121,6 +125,18 @@ def __init__(
         self.dropout = dropout
         self.add_relative_index = add_relative_index
 
+        if norm_type == "LayerNorm":
```

Comment:
How about we also optionally accept a `nn.Module` here, and resolve the string only if `isinstance(self.norm_type, str)` ? That would allow users to also specify their own modules.